lib/get_browsers.es6.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+const https = require('https');
+const env = require('process').env;
+const argv = require('yargs').argv;
+
+module.exports = new Promise(function(resolve, reject) {
+  // if historical is not defined as commandline args,
+  // fetch list of browsers info from envs file
+  if ( !argv.historical ) {
+    const browsers = JSON.parse(
+      fs.readFileSync(`${__dirname}/../${env.SELENIUM_HOST}_envs.json`)
```

Comment:
nit: Please add semi-colon. I recommend setting up ESLint to alert you of missing semi-colons in your editor.